tfhub_dev/assets/sayakpaul/arbitrary-image-stylization-inceptionv3/dr/predict/1.md

```diff
@@ -0,0 +1,29 @@
+# Lite sayakpaul/arbitrary-image-stylization-inceptionv3/dr/predict/1
+TF Lite quantized prediction part of the Arbitrary Style Transfer model.
+
+<!-- parent-model: sayakpaul/arbitrary-image-stylization-inceptionv3/1 -->
+<!-- asset-path: https://github.com/sayakpaul/Adventures-in-TensorFlow-Lite/releases/download/v0.4.0/style_predict_hybrid_tflite.tar.gz -->
+
+[![Open Colab Notebok]](https://colab.research.google.com/github/sayakpaul/Adventures-in-TensorFlow-Lite/blob/master/Style_Transfer_Demo_InceptionV3.ipynb)
```

Comment:
The reason for the validator test failure is because "notebook" should start with lowercase n and at the moment it's capitalized. Example:
`[![Open Colab notebok]](https://...)`
Sorry I missed that before. Same for the rest of the Lite files.

Reply:
@maringeo I will edit it right away.

Reply:
@maringeo should be fixed now.

Reply:
Oops, sorry,I made a typo, it should be notebook, not notebok.

Reply:
My bad I did not catch it :( 

I just pushed the fix.

Reply:
Thanks, all tests pass now. I'll start the import to TF Hub process and let you know once the models are available on tfhub.dev.

Reply:
Fantastic. 

@khanhlvg FYI.

Reply:
Thanks Sayak!
FYI, minor thing, you can use `!pip uninstall -q -y tensorflow` in your notebook to remove the needs of user having to answer "yes/no" in the first cell.

Reply:
Gotcha.

Reply:
> Thanks, all tests pass now. I'll start the import to TF Hub process and let you know once the models are available on tfhub.dev.

@maringeo just saw that the models were imported to TensorFlow Hub: https://tfhub.dev/sayakpaul/lite-model/arbitrary-image-stylization-inceptionv3/dr/predict/1. Thank you :)